cmd/kubeadm/app/phases/certs/certlist.go

```diff
@@ -236,7 +236,7 @@ var (
 	// KubeadmCertRootCA is the definition of the Kubernetes Root CA for the API Server and kubelet.
 	KubeadmCertRootCA = KubeadmCert{
 		Name:     "ca",
-		LongName: "self-signed kubernetes CA to provision identities for other kuberenets components",
+		LongName: "self-signed kubernetes CA to provision identities for other kubernetes components",
```

Comment:
lets make Kubernetes uppercase.

Reply:
I am just fixing a typo, should this change include several changes from `kubernetes` to `Kubernetes`? If I make it uppercase I should be consistent and do it in several places. I think this change is kind of out of the scope for this PR. Maybe I should not fix the typo even on this PR, but is something that caught my eye...

Reply:
kubernetes being lowercase is a typo on it's own because that's how the project name is written. :\
better have it in a separate PR. i will log an issue in k/kubeadm.
thanks.